tests/test_kafka.py

```diff
@@ -103,6 +103,7 @@ def setup_spark(kerberized_kafka, configure_security_spark, configure_universe):
     try:
         utils.upload_dcos_test_jar()
         utils.require_spark()
+        sdk_cmd.run_cli('package install --cli dcos-enterprise-cli --yes')
```

Comment:
It is very strange to me that the installation of `kerberized_kafka` does not install this CLI as it is **required** there to create the secrets storing the Kerberos keytabs.
```suggestion
```

Reply:
removed